### ISSUE TITLE
Group favourite and switch together as one element for VoiceOver users

### ISSUE THREAD

**Issue by sempruijs** [element-hq/element-x-ios#2563]
### Steps to reproduce

Go to the group details page:
1. Open a conversation timeline view
2. click on the group name in the menu bar
You can now see the group details page.

### Outcome

There are 2 elements for VoiceOver users that I think should be grouped into one. ```favorites``` on the left and ```switch button {boo}```. The standard is to group these 2 elements together so that VoiceOver users hear:
```favorite, switch button on```

You can go to the settings app on your iPhone to see an example of this behaviour.

It are all small little details that can make the experience feel great :)

### Operating system version

17.3.1

### Application version

1.6.10 (94)

### Homeserver

matrix.rog

### Will you send logs?

No
